### ISSUE TITLE
Keyboard focus outline broken in Firefox and Safari

### ISSUE THREAD

**Issue by EllyLoel** [google/model-viewer#4221]
### Description

When tabbing focus onto `<model-viewer>` there is no focus outline visible. This is a  [WCAG 2.0: 2.4.7 Focus Visible](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html) failure.

#### Live Demo
https://modelviewer.dev/

### Version
- model-viewer: v3.1.0

### Browser Affected

- [ ] Chrome
- [ ] Edge
- [x] Firefox 113
- [ ] IE (Not sure, can't test as I don't have a Windows PC)
- [x] Safari 167

### OS

- [ ] Android (N/A)
- [ ] iOS (N/A)
- [ ] Linux (Not sure, can't test as I don't have a Linux VM setup ATM)
- [x] MacOS 13.3.1
- [ ] Windows (Not sure, can't test as I don't have a Windows PC)

**Comment by elalish** [comment]
Huh, indeed. I wonder when that broke? Hopefully we just need to update our version of the focus-visible polyfill...

**Comment by elalish** [comment]
Look at that, the polyfill isn't needed anymore! https://github.com/WICG/focus-visible#support
